core/controllers/editor.py

```diff
@@ -286,6 +286,8 @@ def _get_exploration_data(
                 exploration_id, exp_user_data.draft_change_list_exp_version)
             if exp_user_data and exp_user_data.draft_change_list_exp_version
             else None)
+        email_preferences = user_services.get_email_preferences_for_exploration(
```

Comment:
These are not global email preferences. So I think it should be 'exploration_email_preferences' ?